tasks/ebpf.py

```diff
@@ -159,3 +207,99 @@ def print_verification_stats(
 
     with stdout_or_file(out) as f:
         write_verifier_stats(stats_diff, f, jsonfmt)
+
+
+def print_complexity_legend():
+    if colored is None:
+        raise Exit("termcolor is required to print colored output")
+
+    print("\n\n")
+    print("## Complexity legend ##")
+    print("For each source line the verifier processed, we output:")
+    print("  · A code [Ii|Pp|Tt] where:")
+    print("    - Ii is the number of eBPF instructions generated by this source line")
+    print("    - Pp is the number of passes the verifier made over assembly generated by this source line")
+    print("    - Tt is the total number of instructions processed by the verifier for this source line")
```

Comment:
Do we ever use the uppercase versions?

Reply:
The uppercase is meant to represent the number that gets placed but I see how that's confusing. Let me change it.

Reply:
It looks like you kept the uppercase version, but the lowercase one is used in the output. Make sure to change the line starting with `A code` too.

Reply:
Done, let me know if it looks better now.